modules/mono/editor_templates/CharacterBody3D/basic_movement.cs

```diff
@@ -0,0 +1,45 @@
+// meta-description: Classic movement for gravity games (FPS, TPS, ...)
+// meta-default: true
+
+using _BINDINGS_NAMESPACE_;
+using System;
+
+public partial class _CLASS_ : _BASE_
+{
+    public const int Speed = 8;
+    public const int JumpForce = 4;
```

Comment:
This needs to be updated for consistency with the changes I proposed (or whatever ends up being decided).